ws_fnc/AI/fn_createGarrison.sqf

```diff
@@ -224,15 +225,15 @@ for "_x" from 1 to _int do {
 	// Set new variables
 	_u setVariable ["ws_bpos",_bp,true];
 	_b setVariable ["ws_bUnits",_bu + 1,true];
-	_b setVariable ["ws_bPosLeft",_bpl,true]; 
+	_b setVariable ["ws_bPosLeft",_bpl,true];
 
 	if (_debug) then {_mkr = createMarker [format ["%1-bpos",_u],getPos _u];_mkr setMarkerSize [0.5,0.5];_mkr setMarkerType "mil_dot";_mkr setMarkerColor "ColorGreen";};
 };
 
 // If assignGear AI parameter is enabled, set the faction on the spawned units then pass them to assignGear AI.
 if (_assignGearFaction != "") then {
 	[units _grp,_assignGearFaction] call f_fnc_setVirtualFaction;
```

Comment:
Missed this but I think setVirtualFaction needs to be remote execed too. It looks like it just `setVariable`s without the global flag.

Reply:
I would like to make the case that the true answer is to make setVirtualFaction set its variable globally.

Reply:
I don't think we want to change how setVirtualFaction works for now.
Currently it is designed to be run in the unit's init fields which gets executed globally.

Reply:
Adding the global flag for virtualFaction's variable would have no impact on its init field execution, and make it more safe to use elsewhere. But for now I will remoteExec this.